package.json

```diff
@@ -99,9 +97,7 @@
     "serve-static": "^1.14.0",
     "shx": "^0.3.2",
     "sinon": "^12.0.1",
-    "terser-webpack-plugin": "^5.1.1",
     "threads": "^1.6.5",
-    "url-polyfill": "^1.1.5",
```

Comment:
This was added in 097c7b689a6e8b8ddff526897df9b478b139e500 for legacy browser testing and is no longer needed.